includes/wikia/models/DesignSystemCommunityHeaderModel.class.php

```diff
@@ -475,4 +501,61 @@ private function formatLocalNavData( array $items, int $nestingLevel ): array {
 	private function getSpecialPageURL( $name ): string {
 		return SpecialPage::getTitleFor( $name )->getLocalURL();
 	}
+
+	private function isFandomStoreCommunity( $wikiId ) {
```

Comment:
This function seems to be very unnecessary, there's no logic in it and it's only used once.

Reply:
I've removed this function because the logic on whether to hit the API is now in the API Controller.